### DISCUSSION TITLE
Maximale Anzahl der Geräte im Sunny Portal nicht möglich

### DISCUSSION THREAD

**Discussion by Bernd275** [camueller/SmartApplianceEnabler#477] [Q&A]
Ich möchte gerne die max Anzahl der Geräte in SunnyPortal einfügen, leider bietet mit die SunnyPortal Website keine Auswahl Häkchen für alle Geräte an. Habt ihr noch eine Idee, Browser Cache hab ich gelöscht, InternetExplorer und Firefox hab ich versucht.

Kann man den SAE nicht etwas anpassen und den Verbrauertyp für die restlichen Geräte dem SunnyPortal als Energiemonitoring anbieten und nicht als Direkte Kommunikation?

![image](https://user-images.githubusercontent.com/122307990/219966712-2de94883-b436-44d2-b379-97b5aa66b441.png)

**Comment by camueller** [comment]
Für die Reserve-Geräte solltest Du keine Schalter im SAE konfigurieren. Ansonsten kannst Du von SAE-Seite nichts weiter machen, soweit ich mich erinnere.

**Reply by Bernd275** [reply]
ich hab nur dies eingestellt, keine Schalter, keine Zähler, keine Zeitpläne

![image](https://user-images.githubusercontent.com/122307990/219967732-c870f87e-f4d9-4d6b-9947-fa7a0a7a8a6d.png)

**Reply by chubakka0815** [reply]
Ich habe auch das selbe Problem, das ich die Mehrfachauswahl im Sunny Portal nicht habe, egal wie die Reservegeräte konfiguriert sind. Habe auch schon Mal SAE komplett neu konfiguriert und natürlich vorher alle Geräte aus dem Portal gelöscht. Ich vermute Mal das SMA evtl. etwas angepasst hat, was neue Anlagen/ Firmware Stände betrifft.

**Reply by Bernd275** [reply]
Ich hab das gleich auch schon durch SAE komplett neu aufgesetzt und den SMA HM2.0 sogar auf Werksreset und im Sunnyportal alles neu eingerichtet.

Deswegen ja meine Frage ob man beim SAE eventuell die Auswahl der Verbrauchstypen noch mit reinprogrammieren kann. Denn bis jetzt erkennt das Portal alle Geräte als "Direkte Kommunikation"

![image](https://user-images.githubusercontent.com/122307990/220403406-9f477ab1-27a3-470a-97a7-f9ec42ec130f.png)

**Comment by camueller** [comment]
> Deswegen ja meine Frage ob man beim SAE eventuell die Auswahl der Verbrauchstypen noch mit reinprogrammieren kann.

Nein. Ist nicht Bestandteil des SEMP-Protokolls.

**Reply by Bernd275** [reply]
Okay schade, dann wird SMA doch bei ihrem letzten Update ende Januar etwas verändert haben.
